charts/spark-operator-chart/pod-templates/delta-streamer-driver.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Pod
+spec:
+  priorityClassName: delta-streamer-driver-priority-class
```

Comment:
i don't see `priorityClassName` in documentation of 3.2 version , confirm it works on 3.2 
https://spark.apache.org/docs/3.2.0/running-on-kubernetes.html#pod-template

Reply:
tested the flow in staging environment. Updated spark-operator in gcp-acme org, staging namespace. And upgraded `ds-job-multiple-tables-c9a13abf-5aef-4718` delta streamer job. The driver pod that was created got the priority as expected.